.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.5.9, 2.6.7, 2.7.3, 3.0.1]
+        ruby: [2.7.3, 3.0.1]
```

Comment:
```suggestion
        ruby: [2.7.5, 3.0.3]
```

Reply:
@mgrunberg My mistake. It looks like the Docker images have to updated first (https://github.com/rails-sqlserver/activerecord-sqlserver-adapter/runs/4609524313?check_suite_focus=true). Could you revert this change?